cpp/include/cudf/binaryop.hpp

```diff
@@ -202,5 +202,12 @@ cudf::data_type binary_operation_fixed_point_output_type(binary_operator op,
                                                          cudf::data_type const& lhs,
                                                          cudf::data_type const& rhs);
 
+std::unique_ptr<column> binary_operation_compiled(
```

Comment:
Why is `compiled` in the name? I thought the goal was to replace `binary_operation` (JIT based) with compiled binary ops.